DuckDuckGo/PrivacyProtectionController.swift

```diff
@@ -47,7 +47,12 @@ class PrivacyProtectionController: ThemableNavigationController {
             overrideUserInterfaceStyle = .light
         }
 
-        navigationBar.isHidden = isPad
+        if DefaultVariantManager().isSupported(feature: .iPadImprovements) {
```

Comment:
if in experiment let new logic determine if the navigation bar should be hidden, otherwise use the previous logic (which was wrong in some circumstances, but 🤷 )